src/js/actions/transform.js

```diff
@@ -1045,6 +1045,26 @@ define(function (require, exports) {
         return Promise.join(positionPromise, dispatchPromise);
     };
 
+    var beforeStartupCommand = function () {
+        var _transformHandler = function () {
```

Comment:
This is just a drive-by comment, but since I've been looking at the (former) "move" handler: JC made some artboard nesting changes, which I recently tweaked to not fail on option-drag.  Is that logic obsoleted in the new world, or do you  need to implement it in _transformHandler now?  See: https://github.com/adobe-photoshop/spaces-design/pull/1618/files#diff-1bd8d606e837ab0e4ebf7c4eb81890aaR636

Two things for sure: 

1. issue 1613 is not fixed in this branch
1. Dragging across artboards causes history store re-init